Fitness/Controllers/HabitTrackingController.swift

```diff
@@ -538,7 +555,14 @@ extension HabitTrackingController: HabitTrackerOnboardingDelegate {
             if let previousCell = swipedCell {
                 previousCell.swipeRight()
             }
-            habitTableView.separatorStyle = .none
+            
+            for indexPath in habitTableView.indexPathsForVisibleRows ?? [] {
```

Comment:
can we use a `forEach` loop here instead. just a bit  ~ swiftier ~

Reply:
👍🏻